2024-02-02 20:08:22,846:DEBUG:certbot._internal.main:certbot version: 2.8.0
2024-02-02 20:08:22,847:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2024-02-02 20:08:22,847:DEBUG:certbot._internal.main:Arguments: ['--webroot', '--webroot-path=/usr/share/nginx/html/letsencrypt', '--email', 'desarrollo.jadalsoftware@gmail.com', '--agree-tos', '--no-eff-email', '-d', 'migracion.jadal.pe']
2024-02-02 20:08:22,847:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2024-02-02 20:08:22,890:DEBUG:certbot._internal.log:Root logging level set at 30
2024-02-02 20:08:22,891:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2024-02-02 20:08:22,891:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A seperate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x7ff6170a7730>
Prep: True
2024-02-02 20:08:22,892:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x7ff6170a7730> and installer None
2024-02-02 20:08:22,892:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2024-02-02 20:08:23,039:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/1550272667', new_authzr_uri=None, terms_of_service=None), 9b507c65556e201cf96f44fb1f74216d, Meta(creation_dt=datetime.datetime(2024, 2, 2, 17, 5, 57, tzinfo=<UTC>), creation_host='ee38a32c6a85', register_to_eff=None))>
2024-02-02 20:08:23,040:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2024-02-02 20:08:23,044:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2024-02-02 20:08:23,219:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 752
2024-02-02 20:08:23,220:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 20:08:23 GMT
Content-Type: application/json
Content-Length: 752
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.3-September-21-2022.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "p6gSkzdOHsA": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "renewalInfo": "https://acme-v02.api.letsencrypt.org/draft-ietf-acme-ari-02/renewalInfo/",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert"
}
2024-02-02 20:08:23,220:DEBUG:certbot._internal.display.obj:Notifying user: Requesting a certificate for migracion.jadal.pe
2024-02-02 20:08:23,235:DEBUG:acme.client:Requesting fresh nonce
2024-02-02 20:08:23,235:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2024-02-02 20:08:23,292:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2024-02-02 20:08:23,292:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 20:08:23 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 2ksh1KUsfV6pq8rNNCUYs8sSte9s0eLCt3oEYN-YRd_w7liFgkE
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2024-02-02 20:08:23,292:DEBUG:acme.client:Storing nonce: 2ksh1KUsfV6pq8rNNCUYs8sSte9s0eLCt3oEYN-YRd_w7liFgkE
2024-02-02 20:08:23,292:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "migracion.jadal.pe"\n    }\n  ]\n}'
2024-02-02 20:08:23,298:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MDI3MjY2NyIsICJub25jZSI6ICIya3NoMUtVc2ZWNnBxOHJOTkNVWXM4c1N0ZTlzMGVMQ3Qzb0VZTi1ZUmRfdzdsaUZna0UiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "P85-xTkR9ToaoeaXDwVX1vbsY3aF9r5R_8OCC9tTGmULGJ3Xh5Z8zlTy9CP-eHumgsP7f-sa0twfvO8l78MNqL7LIJZjiBxEmpJDt59s2vaQpgpZkY_KPrUzFGmx_8nZMelgtWPggps7B1W-gGi3eDraM83ogVLj2863UCNEp95ZFFtfRGVry-xdlm7SMHQdMgNfqLbM66JnkMUCxDmBacysqWoUwjKsYEHz3NveXTFKCiwZ6FukeRasoQuuEmkr7qhRbCvVlI9Em8vRThrBUhE_UJETmQr9zt3iFuFXHdHRrAHS9rKhmwWqw68sakYPepAXK62fz1VIEGUPDywzCg",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogIm1pZ3JhY2lvbi5qYWRhbC5wZSIKICAgIH0KICBdCn0"
}
2024-02-02 20:08:23,377:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 344
2024-02-02 20:08:23,377:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Fri, 02 Feb 2024 20:08:23 GMT
Content-Type: application/json
Content-Length: 344
Connection: keep-alive
Boulder-Requester: 1550272667
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/1550272667/241360688427
Replay-Nonce: kRyagOpUzmaDZ-cuJasH1oFMjReWmlgDeGQ7Ls80DknTOVP-jLg
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2024-02-09T20:08:23Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "migracion.jadal.pe"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz-v3/310892755367"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/1550272667/241360688427"
}
2024-02-02 20:08:23,377:DEBUG:acme.client:Storing nonce: kRyagOpUzmaDZ-cuJasH1oFMjReWmlgDeGQ7Ls80DknTOVP-jLg
2024-02-02 20:08:23,377:DEBUG:acme.client:JWS payload:
b''
2024-02-02 20:08:23,379:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/310892755367:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MDI3MjY2NyIsICJub25jZSI6ICJrUnlhZ09wVXptYURaLWN1SmFzSDFvRk1qUmVXbWxnRGVHUTdMczgwRGtuVE9WUC1qTGciLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzMxMDg5Mjc1NTM2NyJ9",
  "signature": "K2-aNNqzaZTFbYYNEyNzBulOpsNWEUReBENOVVUrzmhMKJpXUuU1fK9omcJb1abvbSeqWS0xoQebPd7kzNYQYBoa3vruozuQ2YB5TtzCFhQGBkTZzX8Ed59ZGc3OWje2pU1jcBeaFP4Zz19LlWnyBhDERyjjRt1EFUaFeo5eVglZ9kyQzWCaYJhnfCiOeMq3uJJA0tJieyTqeBecYKQX0F7yvYK4qLEk7TWePRkNtBtwmD4ZWHbHFM_zdeuXGiN2-QiMYt-IT-MM9F3D5EYzWsT7874TaTES9ZgMLALn6uo_l3eRmKlAgCpq4hLo62o3WUjQiuc5YYirDaKBuuu3-Q",
  "payload": ""
}
2024-02-02 20:08:23,462:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/310892755367 HTTP/1.1" 200 802
2024-02-02 20:08:23,462:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 20:08:23 GMT
Content-Type: application/json
Content-Length: 802
Connection: keep-alive
Boulder-Requester: 1550272667
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 2ksh1KUsxusaA6-TmvyP-yyx5B9lU5VeF2vhake-qGQiuhg4JwA
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "migracion.jadal.pe"
  },
  "status": "pending",
  "expires": "2024-02-09T20:08:23Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310892755367/3vKQuw",
      "token": "thr9bpGWoFWyyAui97CH3EJFfRMzPjRNd4khHQ-Y8To"
    },
    {
      "type": "dns-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310892755367/0EsZcg",
      "token": "thr9bpGWoFWyyAui97CH3EJFfRMzPjRNd4khHQ-Y8To"
    },
    {
      "type": "tls-alpn-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310892755367/HrA-Gg",
      "token": "thr9bpGWoFWyyAui97CH3EJFfRMzPjRNd4khHQ-Y8To"
    }
  ]
}
2024-02-02 20:08:23,462:DEBUG:acme.client:Storing nonce: 2ksh1KUsxusaA6-TmvyP-yyx5B9lU5VeF2vhake-qGQiuhg4JwA
2024-02-02 20:08:23,463:INFO:certbot._internal.auth_handler:Performing the following challenges:
2024-02-02 20:08:23,463:INFO:certbot._internal.auth_handler:http-01 challenge for migracion.jadal.pe
2024-02-02 20:08:23,463:INFO:certbot._internal.plugins.webroot:Using the webroot path /usr/share/nginx/html/letsencrypt for all unmatched domains.
2024-02-02 20:08:23,463:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge
2024-02-02 20:08:23,465:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge/thr9bpGWoFWyyAui97CH3EJFfRMzPjRNd4khHQ-Y8To
2024-02-02 20:08:23,466:DEBUG:acme.client:JWS payload:
b'{}'
2024-02-02 20:08:23,467:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/chall-v3/310892755367/3vKQuw:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MDI3MjY2NyIsICJub25jZSI6ICIya3NoMUtVc3h1c2FBNi1UbXZ5UC15eXg1QjlsVTVWZUYydmhha2UtcUdRaXVoZzRKd0EiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLXYzLzMxMDg5Mjc1NTM2Ny8zdktRdXcifQ",
  "signature": "l1vfV0mIZZUWUspByZD_fUKxMTMaxcxIj7W446vnQqVXsQK2hRs8YOGQjKQoohoFfShTtSWiFn3kEd6wHglhRFxi1sbGeh0cJ3cdSzcoL_eopHWVhO2LKS0oN61r6W27pTILvcZAPZbRVG34oUxcUTO21E_g_dcLmeqDm2O-PdrN_2pCL2_p2iXEDQoPdN9WK3dAinMXkIIikic5ksa3M7S5oi3CAkQABWleD0OobyrgwYMdtyZsTaOIO03KlmvxYWG5W0Lm1hWRfoQbReX5S6NQ-TbxWz0PkUWKtnCW-hq6h29dlDTjO4UKaberwmehuRt0DCCjRKwl7BW49pGhpA",
  "payload": "e30"
}
2024-02-02 20:08:23,528:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/chall-v3/310892755367/3vKQuw HTTP/1.1" 200 187
2024-02-02 20:08:23,528:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 20:08:23 GMT
Content-Type: application/json
Content-Length: 187
Connection: keep-alive
Boulder-Requester: 1550272667
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-v02.api.letsencrypt.org/acme/authz-v3/310892755367>;rel="up"
Location: https://acme-v02.api.letsencrypt.org/acme/chall-v3/310892755367/3vKQuw
Replay-Nonce: 2ksh1KUs1_E5smeiXlab2Tl_NFxenKjx5EATYXsiK3JklayUKfA
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "status": "pending",
  "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310892755367/3vKQuw",
  "token": "thr9bpGWoFWyyAui97CH3EJFfRMzPjRNd4khHQ-Y8To"
}
2024-02-02 20:08:23,529:DEBUG:acme.client:Storing nonce: 2ksh1KUs1_E5smeiXlab2Tl_NFxenKjx5EATYXsiK3JklayUKfA
2024-02-02 20:08:23,529:INFO:certbot._internal.auth_handler:Waiting for verification...
2024-02-02 20:08:24,532:DEBUG:acme.client:JWS payload:
b''
2024-02-02 20:08:24,572:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/310892755367:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MDI3MjY2NyIsICJub25jZSI6ICIya3NoMUtVczFfRTVzbWVpWGxhYjJUbF9ORnhlbktqeDVFQVRZWHNpSzNKa2xheVVLZkEiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzMxMDg5Mjc1NTM2NyJ9",
  "signature": "xJq65kJAJaSKVpBf8fKlHc2Mc34QNkqA_8dBbHsyXJ_4axF8ZZ-ShyDfbe_kss2EHltn3MFG4WVP6i-dElcrlFROdBoqJSJJEBzIG0_ni3fGriuW8qpGdmk0DTKkKs-CwzJMLPQSRuln47pnGcXL1m1tyE0XXk7DXQrgKL4l1eeQ1nUnB0R4eUrjyiKOUpisV0ErPFZSdP5UyT4uU2v_dfd12WF7Dx_GYCPO89ztXFT_zKNcfikanh6PvcSkz9y2qPwAML_Xc1yNMD6eALweQbTj6VGmKZmc8eZZR0Ubb350jgeqhgw2fguOBkFiE20lVR7yuQDeAl73jEbCUHrTdg",
  "payload": ""
}
2024-02-02 20:08:24,645:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/310892755367 HTTP/1.1" 200 802
2024-02-02 20:08:24,647:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 20:08:24 GMT
Content-Type: application/json
Content-Length: 802
Connection: keep-alive
Boulder-Requester: 1550272667
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 2ksh1KUsm3ORmyrSxdtyBdUoy-hN0caJ-LfhOHmHH4MHRs1M3qw
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "migracion.jadal.pe"
  },
  "status": "pending",
  "expires": "2024-02-09T20:08:23Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310892755367/3vKQuw",
      "token": "thr9bpGWoFWyyAui97CH3EJFfRMzPjRNd4khHQ-Y8To"
    },
    {
      "type": "dns-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310892755367/0EsZcg",
      "token": "thr9bpGWoFWyyAui97CH3EJFfRMzPjRNd4khHQ-Y8To"
    },
    {
      "type": "tls-alpn-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310892755367/HrA-Gg",
      "token": "thr9bpGWoFWyyAui97CH3EJFfRMzPjRNd4khHQ-Y8To"
    }
  ]
}
2024-02-02 20:08:24,647:DEBUG:acme.client:Storing nonce: 2ksh1KUsm3ORmyrSxdtyBdUoy-hN0caJ-LfhOHmHH4MHRs1M3qw
2024-02-02 20:08:27,651:DEBUG:acme.client:JWS payload:
b''
2024-02-02 20:08:27,652:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/310892755367:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MDI3MjY2NyIsICJub25jZSI6ICIya3NoMUtVc20zT1JteXJTeGR0eUJkVW95LWhOMGNhSi1MZmhPSG1ISDRNSFJzMU0zcXciLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzMxMDg5Mjc1NTM2NyJ9",
  "signature": "Z9Y7vKfwYalElwYdKnoV-MonRHBpvaiO0DcIRg5oae1xmGpUzXpal3Ot-O7J_gK3a0IzCcJg1BE873o7xQs15uvXyxu34BWHc4hQ5f_lB5naFjjYZp6zaiiMBP7IuC5botc0MGxzHBg02wf_O0sts3G7oZ_bSnbkdrXjY5gzTLu0ClHIgBeIrs0GeBkadQfDENg3Y7FHTey6UUwsBPUYbCK19lqWH0SpyWicvp6bU7wjLo3VBNk1yju4EFuczerBjyJTjPTIyc2wThJQZWX2TUMNwqSptCceqq3jrE1lFJKP3CD5AFAluuGnmoabQv2t0sI3-Zoh81UMC7_6vriqRQ",
  "payload": ""
}
2024-02-02 20:08:27,713:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/310892755367 HTTP/1.1" 200 1354
2024-02-02 20:08:27,713:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 20:08:27 GMT
Content-Type: application/json
Content-Length: 1354
Connection: keep-alive
Boulder-Requester: 1550272667
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 2ksh1KUsr2_pvihLKfxLSgroXvEJr_mFX24pBZqxEZEAI1D5R7A
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "migracion.jadal.pe"
  },
  "status": "invalid",
  "expires": "2024-02-09T20:08:23Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "invalid",
      "error": {
        "type": "urn:ietf:params:acme:error:unauthorized",
        "detail": "3.85.216.85: Invalid response from https://migracion.jadal.pe/.well-known/acme-challenge/thr9bpGWoFWyyAui97CH3EJFfRMzPjRNd4khHQ-Y8To: 404",
        "status": 403
      },
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310892755367/3vKQuw",
      "token": "thr9bpGWoFWyyAui97CH3EJFfRMzPjRNd4khHQ-Y8To",
      "validationRecord": [
        {
          "url": "http://migracion.jadal.pe/.well-known/acme-challenge/thr9bpGWoFWyyAui97CH3EJFfRMzPjRNd4khHQ-Y8To",
          "hostname": "migracion.jadal.pe",
          "port": "80",
          "addressesResolved": [
            "3.85.216.85"
          ],
          "addressUsed": "3.85.216.85"
        },
        {
          "url": "https://migracion.jadal.pe/.well-known/acme-challenge/thr9bpGWoFWyyAui97CH3EJFfRMzPjRNd4khHQ-Y8To",
          "hostname": "migracion.jadal.pe",
          "port": "443",
          "addressesResolved": [
            "3.85.216.85"
          ],
          "addressUsed": "3.85.216.85"
        }
      ],
      "validated": "2024-02-02T20:08:23Z"
    }
  ]
}
2024-02-02 20:08:27,714:DEBUG:acme.client:Storing nonce: 2ksh1KUsr2_pvihLKfxLSgroXvEJr_mFX24pBZqxEZEAI1D5R7A
2024-02-02 20:08:27,714:INFO:certbot._internal.auth_handler:Challenge failed for domain migracion.jadal.pe
2024-02-02 20:08:27,714:INFO:certbot._internal.auth_handler:http-01 challenge for migracion.jadal.pe
2024-02-02 20:08:27,714:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: migracion.jadal.pe
  Type:   unauthorized
  Detail: 3.85.216.85: Invalid response from https://migracion.jadal.pe/.well-known/acme-challenge/thr9bpGWoFWyyAui97CH3EJFfRMzPjRNd4khHQ-Y8To: 404

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2024-02-02 20:08:27,715:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2024-02-02 20:08:27,715:DEBUG:certbot._internal.error_handler:Calling registered functions
2024-02-02 20:08:27,715:INFO:certbot._internal.auth_handler:Cleaning up challenges
2024-02-02 20:08:27,715:DEBUG:certbot._internal.plugins.webroot:Removing /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge/thr9bpGWoFWyyAui97CH3EJFfRMzPjRNd4khHQ-Y8To
2024-02-02 20:08:27,717:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2024-02-02 20:08:27,718:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
  File "/opt/certbot/src/certbot/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1869, in main
    return config.func(config, plugins)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1600, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 143, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 517, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 428, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 496, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2024-02-02 20:08:27,723:ERROR:certbot._internal.log:Some challenges have failed.
